README.md

```diff
@@ -25,6 +26,32 @@ Other folders in [/src](/src) contain nanoFramework projects that you can refere
 
 **Important:** If you plan to clean up the code in [/src/devices_generated/](/src/devices_generated/), please copy your work to the [/src/devices/](/src/devices/) folder as the content of [/src/devices_generated/](/src/devices_generated/) will be overwritten by the generator tool.
 
+Please check the [detail list of tips and tricks](./tips-tricks.md) to facilitate the migration. The generator takes care of some heavy lifting but there is always some manual adjustments needed.
+
+We are using the following structure for the bindings:

```

Comment:
Haven't been able to fully test it yet but have some code to simplify the solution creation which basically just takes the sln file from dotnet/iot and edits it instead of creating a new one.  I think this will simplify the generator code and mimicking the solution structure may avoid further errors and be easier to compare with dotnet/iot longer term.

Not a hard opinion but just a thought to consider.